Editorial changes for issues #959, #983, #985, #988, #998, #1000, and #1021 (#1030)
moq-wg/moq-transport: draft-ietf-moq-transport.md

* {{model}} describes the data model employed by MOQT.

> * {{session}} covers aspects of setting up an MOQT session.

Reviewer: Use of "a" vs "an" depends on whether you read "MoQT" as "mock-T" or "M-O-Q-T". Similar to "MoQT" vs "MOQT" we should decide a consistent language for the draft and stick with it. I'm inclined to go with "an" (accept changes in this PR) but don't feel that strongly.

Reviewer 2: I'm ambivalent on a vs an, other than we should be consistent.

Application:

> : The entity using MOQT to transmit and receive data.

Reviewer: We should decide whether `MoQT` or `MOQT` is the preferred way to write the acronym. I have been writing `MoQT` this whole time, but both are used inconsistently in the draft, so we probably just need to settle on one.

#### DELIVERY TIMEOUT Parameter {#delivery-timeout}

The DELIVERY TIMEOUT parameter (Parameter Type 0x02) MAY appear in a

> TRACK_STATUS, SUBSCRIBE, SUBSCRIBE_OK or SUBSCRIBE_UDPATE message.

Reviewer: I prefer including the Oxford comma, I don't know if there's an accepted IETF styling for this.

Reviewer 3: I don't think there is, but I think the drafts I've been on recently have also preferred the Oxford comma.

The PATH parameter (Parameter Type 0x01) allows the client to specify the path
of the MoQ URI when using native QUIC ({{QUIC}}).  It MUST NOT be used by

> the server, or when WebTransport is used.  When a PATH parameter is received 
> from a server, or when a PATH parameter is received while WebTransport is used, 
> or when a PATH parameter is received by a server but the server does not 
> support the specified path, the session MUST be closed with Invalid Path.

Reviewer: I think existing wording is a little unclear, but prefer just a small change over suggested change:
```suggestion
the server, or when WebTransport is used.  If the peer receives a PATH
parameter from the server when WebTransport is used, or receives a 
PATH parameter the server does not support, it MUST close the session 
with Invalid Path.
```
